features/google/src/model/analyses/ministories/accesslog-file-summary.js

```diff
@@ -0,0 +1,19 @@
+import { ReportAnalysis } from "@polypoly-eu/poly-analysis";
+import analysisKeys from "../analysisKeys";
+import { convertFileSizeUnit } from "../utils/analysis-tools";
+
+export default class AccessLogSummaryAnalysis extends ReportAnalysis {
+    async analyze({ dataAccount: googleAccount }) {
+        const accessLogSummary = googleAccount.accessLogSummary;
+        const allAccessLogsSummary = {
+            fileName: accessLogSummary[0].fileName,
+            fileSize: convertFileSizeUnit(
```

Comment:
For the report it would also be enough to display the file size directly in bytes. We do the same for example for the archive metadata. This value gets used when we analyse the reports, and it is easier if we get directly the number of bytes. We also use the number of bytes to detect for example identical reports.